src/js/components/FormField/FormField.js

```diff
@@ -261,13 +297,25 @@ const FormField = forwardRef(
           ) {
             wantContentPad = true;
           }
-          if (
+
+          const isInputComponent =
             child &&
             child.type &&
-            grommetInputNames.indexOf(child.type.displayName) !== -1 &&
+            grommetInputNames.indexOf(child.type.displayName) !== -1;
+
+          if (
+            isInputComponent &&
             child.props.plain === undefined &&
             child.props.focusIndicator === undefined
           ) {
+            // Apply the modified focusIndicator
+            if (wantFocusIndicator) {
+              const modifiedFocusIndicator =
+                theme.formField?.focus?.conatinerFocus === false;
+              return cloneElement(child, {
+                focusIndicator: modifiedFocusIndicator,
+              });
+            }
             return cloneElement(child, {
               plain: true,
               focusIndicator: false,
```

Comment:
Could the above "if (wantFocusIndicator) condition be removed and just change the existing return to be:

```
return cloneElement(child, {
              plain: true,
              focusIndicator: theme.formField?.focus?.containerFocus === true ? false : true,
...
```